src/inmanta/data/__init__.py

```diff
@@ -1274,11 +1275,26 @@ class Compile(BaseDocument):
 
     @classmethod
     async def get_unhandled_compiles(cls) -> "List[Compile]":
+        # TODO: is not null? very strange
```

Comment:
this method returns compiles that have completed the compile, but handlers have not been notified